content/en/docs/concepts/workloads/controllers/replicationcontroller.md

```diff
@@ -98,7 +98,7 @@ Pods Status:    3 Running / 0 Waiting / 0 Succeeded / 0 Failed
 To list all the pods that belong to the ReplicationController in a machine readable form, you can use a command like this:
 
 ```shell
-pods=$(kubectl get pods --selector=app=nginx --output=jsonpath={.items..metadata.name})
+pods=$(kubectl get pods --selector=app=nginx --output=jsonpath={.items.metadata.name})
```

Comment:
This command is correct, `..`  is used here for recursive descent. I tried this one and it gave me the expected output:

```
$ kubectl get pods --selector=app=nginx --output=jsonpath={.items..metadata.name}
nginx-8rhh6 nginx-vsm8q nginx-vv9bq
```